app/webpacker/components/Media/MediaTable.jsx

```diff
@@ -0,0 +1,254 @@
+import React, { useEffect, useState } from 'react';
+import _ from 'lodash';
+import { Table, Confirm, Modal, Button } from 'semantic-ui-react';
+import { competitionUrl, apiV0Urls } from '../../lib/requests/routes.js.erb';
+import { countries, continents, years } from '../../lib/wca-data.js.erb';
+import { dateRangeBetween } from '../../lib/helpers/media-table';
+import useLoadedData from '../../lib/hooks/useLoadedData';
+import useSaveAction from '../../lib/hooks/useSaveAction';
+import { DateTime } from 'luxon';
+import {
+  DropdownMenu,
+  DropdownItem,
+  DropdownHeader,
+  DropdownDivider,
+  Dropdown,
+} from 'semantic-ui-react'
+const countryOptions = _.map(countries.byIso2, (country) => ({
```

Comment:
super nit: would be good to see some newlines in between these variable declarations.